### PR TITLE
Fix/lagging event detail issues

### PR DESCRIPTION
**Pull request by sieuhuflit**
## Overview

- Fix lagging event detail issues

Reference issue: https://github.com/flutter/flutter/issues/32804

### CODE REVIEW

File: lib/core/presentation/widgets/common/app_backdrop/app_backdrop.dart [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    this.strength = 1,
+    this.child,
+  }) : super(key: key);
+
+  final double strength;
+  final Widget? child;
+
+  @override
+  Widget build(BuildContext context) {
+    final double normalStrength = clampDouble(strength, 0, 1);
```
Comment (sieuhuflit) [inline lib/core/presentation/widgets/common/app_backdrop/app_backdrop.dart:18]: Add this to force maximum strength

Refer from : https://github.com/gskinnerTeam/flutter-wonderous-app/blob/main/lib/ui/common/app_backdrop.dart

File: ios/Podfile.lock [hunk 2]
```diff
@@ -160,6 +160,23 @@ PODS:
     - GoogleUtilities/Environment (~> 7.7)
     - nanopb (< 2.30910.0, >= 2.30908.0)
     - PromisesObjC (< 3.0, >= 1.2)
+  - GoogleMLKit/BarcodeScanning (4.0.0):
```
Comment (sieuhuflit) [inline ios/Podfile.lock:163]: Latest code seem like missing these Podfile update

File: lib/core/presentation/pages/event/event_detail_page/guest_event_detail_page/widgets/guest_event_poap_offer_item.dart [hunk 3]
```diff
@@ -101,22 +102,31 @@ class GuestEventPoapOfferItemState extends State<GuestEventPoapOfferItemView>
...
+                        ),
+                        const AppBackdrop(),
+                      ],
                     ),
                   ),
                 ),
               ),
             ),
-            const _Blur(),
+            // AppBackdrop(),
```
Comment (mthinh) [inline lib/core/presentation/pages/event/event_detail_page/guest_event_detail_page/widgets/guest_event_poap_offer_item.dart:129]: not sure why we commented it here ?

**Review by mthinh — APPROVED** [review]

File: lib/core/presentation/pages/event/event_detail_page/guest_event_detail_page/widgets/guest_event_poap_offer_item.dart (hunk 3 above)
Comment (sieuhuflit) [reply lib/core/presentation/pages/event/event_detail_page/guest_event_detail_page/widgets/guest_event_poap_offer_item.dart:129]: Ah, it's redundant, I will removed it later 👍

**Pull request merged**